web-scraping-with-scrapy-and-mongodb/books/tests/test_book.py

```diff
@@ -0,0 +1,72 @@
+import unittest
+from pathlib import Path
+
+from books.items import BooksItem
+from books.spiders.book import BookSpider
+from scrapy.http import HtmlResponse, Request
+
+
+def _get_sample_html_content():
+    html_file_path = Path(__file__).parent / "sample.html"
+    with html_file_path.open() as html_file:
+        return html_file.read()
```

Comment:
Here's a small suggestion, but feel free to ignore it.

Since you already use `Path`, you can write this more compactly:

```suggestion
    return html_file_path.read_text("utf-8")
```